client/components/QuestionsAndAnswers/modal/PhotoModal.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { ModalShadow, Modal, PopupImg } from '../styles/Modal.style';
+
+const PhotoModal = ({ currPhoto, handleImgModal }) => (
+  ReactDOM.createPortal(
+    <>
+      {console.log('what is currPhoto', currPhoto)}
+      <ModalShadow />
+      <Modal>
+        <PopupImg
```

Comment:
It looks like right now you exit the modal by clicking the photo. This is opposite from my photo modal and it would be a nice to have if our modals were more standardized -- consider this tech debt and don't worry about this at all until we have an MVP!